src/bootstrap.tsx

```diff
@@ -93,20 +101,25 @@ const useInitialize = () => {
   return {
     isReady,
     libJwt,
+    isFedRampReady,
   };
 };
 
 const App = () => {
   const modules = useSelector(({ chrome }: ReduxState) => chrome?.modules);
   const scalprumConfig = useSelector(({ chrome }: ReduxState) => chrome?.scalprumConfig);
   const documentTitle = useSelector(({ chrome }: ReduxState) => chrome?.documentTitle);
-  const { isReady, libJwt } = useInitialize();
+  const { isReady, libJwt, isFedRampReady } = useInitialize();
 
   useEffect(() => {
     const title = typeof documentTitle === 'string' ? `${documentTitle} | ` : '';
     document.title = `${title}console.redhat.com`;
   }, [documentTitle]);
 
+  if (isFedRampReady && modules && scalprumConfig && fedRampEnv) {
```

Comment:
I think we want to slightly enhance the condition so we can also get the placeholder component in.

```
if(fedRampEnv) {
  return isFedRampReady && modules && scalprumConfig ? <RootApp config={scalprumConfig} /> : <AppPlaceholder />
}
```